docs/jwst/ipc/reference_files.rst

```diff
@@ -1,26 +1,56 @@
-Reference File Types
-====================
-The IPC deconvolution current step uses an IPC reference file.
+Reference Files
+===============
+The IPC deconvolution step uses an IPC reference file.
 
-.. include:: ../includes/standard_keywords.rst
+IPC Reference File
+------------------
+
+:RETYPE: IPC
```

Comment:
Should be REFTYPE?

Reply:
Yep.  That's a typo that's in my template that I've been using to make updates for _all_ the steps and you're the first one to spot it. Good eye!